### PR TITLE
adding pom files and sc of it working

### PR DESCRIPTION
**Pull request by amboraes1**
adding pom files, and a sc of it working

### CODE REVIEW

**Comment by manuelq12** [comment]
products-list.page.ts, shoping-cart.page.ts, login.page.ts, address-step.page.ts, shipping-step.page.ts y payment-step.page.ts 

Hey man, you need to divide the flow in this different pages as specified in the workshop

File: cypress/integration/buy-shirt.spec.ts [hunk 1]
```diff
@@ -1,7 +1,13 @@
 /// <reference types="Cypress" />
 
+import { MenuContentPage, CheckoutPage, PLPPage } from "../page/index";
```
Comment (manuelq12) [inline cypress/integration/buy-shirt.spec.ts]: There should be more pages

products-list.page.ts, shoping-cart.page.ts, login.page.ts, address-step.page.ts, shipping-step.page.ts y payment-step.page.ts

**Review by manuelq12 — CHANGES_REQUESTED** [review]

File: cypress/integration/buy-shirt.spec.ts (hunk 1 above)
Comment (amboraes1) [reply cypress/integration/buy-shirt.spec.ts]: Pages added

Comment (amboraes1) [reply cypress/integration/buy-shirt.spec.ts]: Do I have to add the step to the name?

**Review by manuelq12 — CHANGES_REQUESTED** [review]
Why do you have this here? If there are actions that need to be mande before they should be part of the Pages!

If not, you should erased them and use only the pages

**Review by manuelq12 — CHANGES_REQUESTED** [review]
Lets make it interesting, i want you to use Gherkin for this first test, its important that you use correctly the describe and it. Remember describe is for grouping the test strategy (steps); it is for assertions

i.e
describe('Given a shopping website', ()=>{});
it('Then the t-shirt order must be completed', async () => {});

As in lower steps you'll use AAA, this will be a good practice, to organize your tests

File: cypress/page/cart.page.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/// <reference types="Cypress" />
+
+class CartPage {
+
+    public proceedToCheckoutButton(): void {
+        cy.get(".cart_navigation span").click();
+    }
+}
+
+export { CartPage }
```
Comment (manuelq12) [inline cypress/page/cart.page.ts]: EOF

File: cypress/page/confirmpayment.page.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+/// <reference types="Cypress" />
+
+class ConfirmPaymentPage {
+    public confirmOrderButton(): void {
+        cy.contains("span", "I confirm my order").click();
+    }
+}
+
+export { ConfirmPaymentPage }
```
Comment (manuelq12) [inline cypress/page/confirmpayment.page.ts]: EOF

File: cypress/page/menu-content.page.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    public visitMenuContentPage(): void {
+        cy.visit(this.menuContentPageURL);
+    }
+
+    public goToTShirtMenu(): void {
+        cy.get(this.tShirtMenu).click();
+    }
+
+}
+export { MenuContentPage }
```
Comment (manuelq12) [inline cypress/page/menu-content.page.ts]: EOF

File: cypress/page/orderconfirmation.page.ts [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+class OrderConfirmationPage {
+
+    public orderPlacedTitle(): void {
+        cy.get("#center_column > div > p > strong")
+      .should("have.text", "Your order on My Store is complete.");
+    }
+
+}
+
+export { OrderConfirmationPage }
```
Comment (manuelq12) [inline cypress/page/orderconfirmation.page.ts]: EOF

File: cypress/page/payment.page.ts [hunk 6]
```diff
@@ -0,0 +1,9 @@
+/// <reference types="Cypress" />
+
+class PaymentPage {
+    public bankwireOption(): void {
+        cy.get(".bankwire").click();
+    }
+}
+
+export { PaymentPage }
```
Comment (manuelq12) [inline cypress/page/payment.page.ts]: EOF

File: cypress/page/plp.page.ts [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+    public addToCartButton(): void {
+        cy.get(" div.button-container > a.button.ajax_add_to_cart_button").click();
+    }
+
+    public proceedToCheckoutButton(): void {
+        cy.get("a[title='Proceed to checkout']").click();
+    }
+}
+
+export { PLPPage }
```
Comment (manuelq12) [inline cypress/page/plp.page.ts]: EOF

Why this name, I think this isn't one of the names given in the workshop

File: cypress/page/shipping.page.ts [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+    public tandcCheckox(): void {
+        cy.get("#cgv").click();
+    }
+
+    public proceedToCheckoutButton(): void {
+        cy.contains("button","Proceed to checkout").click();
+    } 
+}
+
+export { ShippingPage }
```
Comment (manuelq12) [inline cypress/page/shipping.page.ts]: EOF

File: cypress/page/signin.page.ts [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+    
+    public passwordInput(password: string): void {
+        cy.get("#passwd").type(password);
+    }
+
+    public signInButton(): void {
+        cy.get("#SubmitLogin").click();
+    }
+}
+export { SigninPage }
```
Comment (manuelq12) [inline cypress/page/signin.page.ts]: EOF

**Review by manuelq12 — CHANGES_REQUESTED** [review]

File: cypress/integration/buy-shirt.spec.ts (hunk 1 above)
Comment (manuelq12) [reply cypress/integration/buy-shirt.spec.ts]: I don't understand the question, can you clarify to me please 😄 ?

**Comment by manuelq12** [comment]
FYI
https://www.quora.com/Why-is-it-good-practice-to-add-a-newline-to-the-end-of-a-file

File: cypress/page/plp.page.ts (hunk 7 above)
Comment (amboraes1) [reply cypress/page/plp.page.ts]: PLP = product listing page, and in the workshop there is no name suggested actually

**Review by manuelq12 — APPROVED** [review]

**Pull request merged**